tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -79,7 +79,9 @@ def _run(self):
                         lock.acquire(lock_ttl=60)
                 except etcd.EtcdLockExpired:
                     continue
-                gevent.sleep(10)
+                gevent.sleep(
+                    int(NS.config.data.get("sync_interval", 10))
```

Comment:
This is not the only place that should use `sync_interval`

All the out of band object .save() with ttl should use ttl value by adding 10-15 seconds to `sync_interval`